addons/project/migrations/9.0.1.1/post-migration.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenUpgrade module for Odoo
+#    @copyright 2014-Today: Odoo Community Association, Microcom
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import logging
+from openupgradelib import openupgrade
+logger = logging.getLogger('OpenUpgrade')
+
+# copied from pre-migration
+column_copies = {
+    'project_task': [
+        ('description', None, None),
+    ],
+}
+
+
+def map_priority(cr):
+    openupgrade.map_values(
+        cr,
+        openupgrade.get_legacy_name('priority'),
+        'priority',
+        [('2', '1')],
+        table='project_task', write='sql')
+
+
+def map_template_state(cr):
+    openupgrade.map_values(
+        cr,
+        openupgrade.get_legacy_name('state'),
+        'state',
+        [('template', 'draft')],
+        table='project_project', write='sql')
+
+
+@openupgrade.migrate()
+def migrate(cr, version):
+    map_priority(cr)
+    map_template_state(cr)
+    for table_name in column_copies.keys():
+        for (old, new, field_type) in column_copies[table_name]:
```

Comment:
Here you can save the loop if you want, as there is only one table and only one field with this case, and makes it more readable.